src/api/environments.ts

```diff
@@ -66,7 +66,7 @@ export async function createEnvironmentFromSuffix(
     form: true,
     body: {
       UniqueName: uniqueName,
-      DomainSuffix: domainSuffix,
+      DomainSuffix: domainSuffix || undefined,
```

Comment:
You've said that this can be `undefined` here, but the function signature says that it has to be a string and that leads all the way back to where it's called from initially.

However the underlying implication here is that the empty string is production. I'm not sure I like that empty string has such significance. Might it be more obvious to developers to make the string "production" refer to the production/suffix-less environment and just check at this stage whether the `domainSuffix === "production"`. 

Both ways we have a magic string (either "" or "production") but at least "production" is a bit more self documenting. What do you think?

Reply:
I have to use "" here or alternatively undefined for the domain suffix because "production" could be a valid domain suffix that someone created through the API. 

This line specifically only sets `DomainSuffix` to `undefined` because otherwise `DomainSuffix=""` will be passed to the REST API which will make it fail. 

The alternative would be to allow for this module to support `undefined` as DomainSuffix rather than `""` but then I would have to add a significantly bigger amount of checks across the entire code base for undefined. 

I could rewrite this line to be: 
`DomainSuffix: domainSuffix.length > 0 ? domainSuffix : undefined` because that's ultimately what I'm doing here

Reply:
So, production is no domain suffix but production could also be the suffix "production"?

Seems like it might get confusing. Perhaps my problem is not with this PR (if it's a blank string then it needs to be set to `undefined` to make the API happy), but with the `twilio-run` side of things. The `serverless-api` module doesn't really hold opinions, however `twilio-run` does. That is probably a better place to try to define what "production" means in the context of `twilio-run` especially if it is going to have a `--production` flag.

What do you think?